cmd/ipfs-cluster-service/main.go

```diff
@@ -381,12 +412,18 @@ this state from disk.  This command renames cluster's data folder to <data-folde
 deprecated data folders to <data-folder-name>.old.<n+1>, etc for some rotation factor before permanatly deleting 
 the mth data folder (m currently defaults to 5)
 `,
+					Flags: []cli.Flag{
+						cli.BoolFlag{
+							Name:  "force, f",
+							Usage: "forcefully proceed with rotating peer state without prompting",
```

Comment:
Same as `import` prompt is still present after check of the `force` flag.

Reply:
This one as well appears only if `-f` is not set.